app/models/facility.rb

```diff
@@ -20,10 +20,20 @@ class Facility < ApplicationRecord
   has_many :blood_pressures, through: :encounters, source: :blood_pressures
   has_many :blood_sugars, through: :encounters, source: :blood_sugars
   has_many :patients, -> { distinct }, through: :encounters
-  has_many :hypertension_patients, -> { distinct }, through: :blood_pressures, source: :patient
   has_many :prescription_drugs
   has_many :appointments
-  has_many :registered_patients, class_name: "Patient", foreign_key: "registration_facility_id"
+
+  has_many :registered_patients,
+           class_name: "Patient",
+           foreign_key: "registration_facility_id"
+  has_many :registered_diabetes_patients,
+           -> { diabetes_only },
```

Comment:
Using the `only` scope here ensures that a patient can't show up in both `registered_diabetes_patients` _and_ `registered_hypertension_patients`?

This asymmetry seems a bit odd to me. Should this just be `-> { with_diabetes }`?

Reply:
This is just a bad merge. It's supposed to be `with_diabetes`. Hopefully the test should break any time now...